core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -569,14 +569,14 @@ enum class UniqueType(val text:String, vararg targets: UniqueTarget, val flags:
     //endregion
 
     ///////////////////////////////////////////// META /////////////////////////////////////////////
-    
-    
-    HiddenWithoutReligion("Hidden when religion is disabled", UniqueTarget.Unit, UniqueTarget.Building, UniqueTarget.Ruins),
-    HiddenBeforePantheon("Hidden before founding a Pantheon", UniqueTarget.Ruins),
-    HiddenAfterPantheon("Hidden after founding a Pantheon", UniqueTarget.Ruins),
-    HiddenAfterGreatProphet("Hidden after generating a Great Prophet", UniqueTarget.Ruins),
+
     AvailableAfterCertainTurns("Only available after [amount] turns", UniqueTarget.Ruins),
-    HiddenWithoutVictoryType("Hidden when [victoryType] Victory is disabled", UniqueTarget.Building, UniqueTarget.Unit),
+    HiddenWithoutReligion("Hidden when religion is disabled", UniqueTarget.Unit, UniqueTarget.Building, UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenBeforePantheon("Hidden before founding a Pantheon", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenAfterPantheon("Hidden after founding a Pantheon", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenAfterGreatProphet("Hidden after generating a Great Prophet", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
```

Comment:
impo these should also be visible to the user

Reply:
I think I'm inclined to agree. But that would also take this PR out of "linting" territory into changing existing behaviour, and I think it already has enough not-quite-related changes mixed in.

I'll take a closer look at what they're actually used for, whether it might make more sense to say "Unavailable" instead of "Hidden" if they'll be user-visible, etc.

Reply:
Right, unhid. But I haven't found anywhere those Uniques would be displayed in the UI anyway, and if a detailed "Ruin Rewards" Civilopedia section or something is added at some point then I think they should probably say "Unavailable" instead of "Hidden".

Reply:
The current ""Ruin Rewards" Civilopedia" is only Improvements - Ancient ruins, TileImprovements.kt line 166+... And doesn't show text from Uniques?

Reply:
AFAICT It just shows the effects/names:

![image](https://user-images.githubusercontent.com/37680486/150701179-633967a9-c5a1-4d26-9b3b-d0063a503d67.png)